lesson-5/Dooley - Lesson 5/style.css

```diff
@@ -0,0 +1,36 @@
+body {
+  font-family: "Work Sans", sans-serif;
+
+}
+
+img {
+	margin-bottom: -4px;
+	padding: 0px;
+}
+.container {
+	width: 300px;
+	border: 0px;
+	margin: 0px;
+}
+
+
+.text {
+	background-color: whitesmoke;
+	padding: 20px 30px 30px 30px;
+	margin: 0px;
+}
+
+.link {
+	margin-top: 30px;
+	margin-bottom: 10px;
+}
+a {
+	color: white;
+	font-weight: bold;
+	text-align: center;
+	background-color: #f96266;
+	text-decoration: none;
+	padding: 10px 100px;
+	border-radius: 10px;
+}
+margin: 20px;
```

Comment:
Did this end up in the wrong place? Or maybe it was left over from some experimenting :)